### PR TITLE
DOC: Small improvements in optimize._lsq docstrings

### PR DESCRIPTION
**Pull request by nmayorov**
Hi, I tried to fix some problems left with docstring after my PRs with least-squares algorithms were merged.

### CODE REVIEW

File: scipy/optimize/_lsq/least_squares.py [hunk 1]
```diff
@@ -231,8 +231,7 @@ def least_squares(
         loss_scale=1.0, scaling=1.0, diff_step=None, tr_solver=None,
         tr_options={}, jac_sparsity=None, max_nfev=None, verbose=0, args=(),
         kwargs={}):
-    """Minimize the robustified sum of squares of nonlinear functions, subject
-    to bound constraints on independent variables.
+    """Robust nonlinear least squares with bounds on the variables.
```
Comment (dlax) [inline scipy/optimize/_lsq/least_squares.py]: Not having an action verb in the summary line is quite unusual. Is this something somebody else suggested?
If you don't like "minimize", maybe "solve" would be better?

Suggestion: "Solve a nonlinear least squares problem with bounds on the variables." Of course, there's no mention of "robust" as this does not fit, is this a big deal if this does not appear here?

File: scipy/optimize/_lsq/lsq_linear.py [hunk 2]
```diff
@@ -33,12 +33,11 @@ def prepare_bounds(bounds, n):
 
 def lsq_linear(A, b, bounds=(-np.inf, np.inf), method='trf', tol=1e-10,
                lsq_solver=None, lsmr_tol=None, max_iter=None, verbose=0):
-    r"""Solve a linear least-squares problem subject to bound constraints on
-    independent variables.
+    r"""Linear least squares with bounds on the variables.
```
Comment (dlax) [inline scipy/optimize/_lsq/lsq_linear.py]: Same here: no verb.

**Comment by dlax** [comment]
Apart from the two comments about verb in summary lines, this looks good to me.

**Comment by nmayorov** [comment]
@dlax, good suggestions, I updated one-line summaries.

**Pull request merged**